### PR TITLE
Change in first Tab

### PR DESCRIPTION
**Pull request by tweddielin**
- Made the prediction table sortable
- Redo all the plots in react-vis
- Made all plots zoomable
- Fixed a babel bug of not recognizing es6

### CODE REVIEW

File: frontend/components/highlight-area/component.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        left: startLoc
+      }
+    }
+  }
+
+  onParentMouseDown(e) {
+    const { marginLeft, innerHeight, onBrushStart } = this.props
+    const location = e.nativeEvent.offsetX - marginLeft
+
+    // TODO: Eventually support drawing as a full rectangle, if desired. Currently the code supports 'x' only
```
Comment (thcrock) [inline frontend/components/highlight-area/component.js]: Can we track TODOs as github issues instead? TODOs still sneak their way into my code from time to time, but I'm trying to stop doing that.

Comment (tweddielin) [inline frontend/components/highlight-area/component.js]: yeah!

File: frontend/components/highlight-area/component.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      }
+    }
+  }
+
+  render() {
+    const { marginLeft, marginTop, innerWidth, innerHeight, color, opacity } = this.props
+    const { drawArea: { left, right, top, bottom } } = this.state
+
+    return (
+      <g transform={`translate(${marginLeft}, ${marginTop})`}
```
Comment (thcrock) [inline frontend/components/highlight-area/component.js]: What is `g`?

Comment (tweddielin) [inline frontend/components/highlight-area/component.js]: SVG element to group SVG shapes together.

File: webapp/controller.py [hunk 3]
```diff
@@ -253,8 +255,8 @@ def get_roc(model_id, evaluation_start_time):
     query_arg = {'model_id': model_id, 'evaluation_start_time': evaluation_start_time}
     pred = query.get_model_prediction(query_arg)
     fpr, tpr, threshold = roc_curve(pred['label_value'], pred['score'])
-    output = [{'key': 'roc', 'values': list(zip(fpr, tpr))},
-              {'key': 'random', 'values': [[0, 0], [1, 1]]}]
+    output = [{'title': 'roc', 'color': '#1f77b4', 'data': [{'x': f, 'y': t} for f, t in zip(fpr, tpr)]},
```
Comment (thcrock) [inline webapp/controller.py]: Should we be deciding color on the backend? This feels like it should be in a stylesheet somewhere, or at least decided on the frontend.

**Pull request merged**